elliott/tests/test_snapshot_cli.py

```diff
@@ -180,14 +188,14 @@ async def test_run_happy_path(self, mock_runtime, mock_konflux_client_init, mock
             "config": {
                 "config": {
                     "Labels": {
-                        "com.redhat.component": "test-component",
+                        "com.redhat.component": "test-component-container",
                         "version": f"v{self.major}.{self.minor}.0",
                         "release": "assembly.test.el8",
                     },
                 },
             },
         }
-        expected_nvrs = [f"test-component-v{self.major}.{self.minor}.0-assembly.test.el8"]
+        expected_nvrs = [f"test-component-container-v{self.major}.{self.minor}.0-assembly.test.el8"]
```

Comment:
Not specific to this line, but i don't see a test for creating multiple snapshots. can you add that?